src/services/export-service.js

```diff
@@ -38,31 +38,22 @@ class ExportService {
     return this.documentStore.find(filter, { sort: lastUpdatedFirst, projection: exportableDocumentsProjection });
   }
 
-  async getDocumentExport({ key, afterRevision, toRevision }) {
+  async getDocumentExport({ key, toRevision }) {
     const revisions = (await this.documentService.getAllDocumentRevisionsByKey(key)).sort(by(d => d.order));
-
-    let nextRevisionIndex;
     const lastRevisionIndex = revisions.findIndex(revision => revision._id === toRevision);
```

Comment:
I think the toRevision is also obsolete. 
The importer just says, "give me what you can export", and based on that it calculates what is to import, reimport and update. 
One caveat is that another revision can be inserted while the user is looking at the import screen, but I'm not sure that that just doesn't save him the time to do another import :). We currently don't allow you to import up to some revision as far as I know.

Reply:
I specifically did not want to remove this because:
1. I see the import as a snapshot, "give me this instance of the documents"
2. We distinguish between "update" and "reimport" and this flag is what facilitates that